sw/airborne/subsystems/ahrs/ahrs_float_dcm.c

```diff
@@ -181,7 +181,7 @@ void ahrs_dcm_update_gps(struct GpsState *gps_s UNUSED)
     ahrs_dcm.gps_age = 0;
     ahrs_dcm.gps_speed = gps_s->speed_3d / 100.;
 
-    if (gps_s->gspeed >= 500) { //got a 3d fix and ground speed is more than 5.0 m/s //FIXME: Should be settable
+    if (gps_s->gspeed >= MIN_COURSE_SPEED) { //got a 3d fix and ground speed is more than MIN_COURSE_SPEED/100 m/s
```

Comment:
I suggest a name like `AHRS_MIN_COURSE_SPEED` or `FLOAT_DCM_MIN_COURSE_SPEED` to make it clear in airframe file that it is related to ahrs filter.
Also, it is probably nicer to have standard units (so m/s): `gps_s->gspeed >= 100 * XXX_MIN_COURSE_SPEED`.